openedx/features/course_experience/templates/course_experience/course-sock-fragment.html

```diff
@@ -57,7 +57,7 @@ <h4>${_('edX Learner Stories')}</h4>
                         </div>
                     </div>
                 % endif
-                <img class="mini-cert" src="${static.url('course_experience/images/verified-cert.png')}"/>
+                <img class="mini-cert" alt="Example Certificate Image" src="${static.url('course_experience/images/verified-cert.png')}"/>
                 <a href="${upgrade_url}">
```

Comment:
@HarryRein any reason why we have a button and a link here?  Reminds me of the Karate Kid: https://youtu.be/0lsIGZKShc8?t=14  You either Karate-do-link, or karate-do-button.  Karate-do-link-and-button; sooner or later, squish like grape.  Outside of the scope of this ticket, but must be fixed.  I'm surprised its not creating more problems as ```<a>``` elements must not have interactive children:
http://w3c.github.io/html/textlevel-semantics.html#the-a-element
Content model:
    Transparent, but there must be no interactive content or a element descendants.

Reply:
Yeah, not sure how that got through either. Seems like we have a standard here of doing a link element around a div, I wonder if the fix is as simple as that - may require some extra styling though. I'm going to merge this PR and we can add a follow on for that.